src/ralph/configuration_management/models.py

```diff
@@ -0,0 +1,35 @@
+from dj.choices import Choices
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from ralph.assets.models import BaseObject
+from ralph.lib.mixins.models import TimeStampMixin
+from ralph.lib.permissions import PermByFieldMixin
+
+
+class SCMScanStatus(Choices):
+    _ = Choices.Choice
+
+    ok = _("ok")
+    fail = _("fail")
+    error = _("error")
+
+
+class SCMScan(
+    PermByFieldMixin,
+    TimeStampMixin,
+    models.Model
+):
+    """Represents software configuration management scan."""
+
+    base_object = models.OneToOneField(BaseObject, on_delete=models.CASCADE)
+    last_scan_date = models.DateTimeField(
```

Comment:
Perhaps it makes change to get rid of the word Scan for this thing. @mkurek, @misiek08 what do you think, if I call it SCMStatusCheck with the attributes called last_checked and check_result. Possible values for check_result could be scm_ok, smc_error, check_failed.